woocommerce-gateway-ebanx/includes/class-wc-ebanx-credit-card-br-gateway.php

```diff
@@ -33,6 +33,15 @@ public function is_available()
     }
 
     /**
+	 * Check if the currency is processed by EBANX
+	 * @param  string $currency Possible currencies: BRL
+	 * @return boolean          Return true if EBANX process the currency
+	 */
+	public function ebanx_process_merchant_currency($currency) {
```

Comment:
Method defined 3 times in similar classes, can't it be moved to the abstract class-wc-ebanx-gateway.php class?